service/builder/sampling_builder.go

```diff
@@ -159,20 +161,16 @@ func (sCfg *SamplingCfg) InitFromViper(v *viper.Viper) *SamplingCfg {
 
 // TailBasedCfg holds the configuration for tail-based sampling.
 type TailBasedCfg struct {
```

Comment:
nit: config as a name should be consistent with other processors.

Reply:
Will rename it when I removed the old config and moved it to tailsampling/config.go (in the follow-up PR).

Reply:
Sounds good, that's good to know.